docs/atsd_install.md

```diff
@@ -0,0 +1,162 @@
+```
+
+```
+sudo docker run \
+  --volume=/:/rootfs:ro \
+  --volume=/var/run:/var/run:rw \
+  --volume=/sys:/sys:ro \
+  --volume=/var/lib/docker/:/var/lib/docker:ro \
+  --volume=/home/user:/root:ro \
+  --publish=8080:8080 \
+  --detach=true \
+  --name=cadvisor \
+  --link atsd:atsd \
+  axibase/cadvisor:latest \
+  --storage_driver=atsd \
+  --storage_driver_user=user \
+  --storage_driver_password=secret-pwd \
+  --storage_driver_atsd_config_path="/root/cadvisor.toml"
+```
+
+#### More options:
+
+key                                      | default value              | description
+-----------------------------------------|----------------------------|------------
+storage_driver_buffer_duration           |1m                          | time for which data is accumulating in a buffer before send
+storage_driver_atsd_url                  |""                          | atsd http/https endpoint
+storage_driver_atsd_write_host           |""                          | tcp/udp destination host:port
+storage_driver_atsd_write_protocol       |"http/https"                | write protocol. Possible settings: http/https, udp, tcp
+storage_driver_atsd_store_major_numbers  |false                       | include statistics for devices with all available major numbers
+storage_driver_atsd_property_interval    |1m                          | container property update interval. Should be >= housekeeping_interval
+storage_driver_atsd_sampling_interval    |housekeeping_interval value | series sampling interval. Should be >= housekeeping_interval
+storage_driver_atsd_docker_host          |"docker-host"               | hostname of the docker machine (entity prefix)
+storage_driver_atsd_store_user_cgroups   |false                       | include statistics for "user" cgroups 
+storage_driver_atsd_buffer_limit         |1000000                     | max series commands to store in buffer until flush
+storage_driver_atsd_connection_limit     |1                           | tcp connection count
+storage_driver_atsd_config_path          |""                          | path to ATSD storage driver config file
+
+You can view the collected metrics under the Entity and Metrics tabs in ATSD.
+> Note that disk metrics are only collected from containers that have attached volumes.
+
+#### Axibase Time Series Database cAdvisor Portals
+
+Default visualization portals for cAdvisor are included in ATSD by default.
+
+Using the built-in Overview, Disk Detail, Host and Multi-Host visualization portals, you can quickly identify bottlenecks in your microservices infrastructure.
+
+#### cAdvisor Overview Portal
+![cAdvisor Overview Portal](http://axibase.com/wp-content/uploads/2015/01/cadvisor_overview_portal2.png)
+
+#### cAdvisor Host Portal
+![cAdvisor Host Portal](http://axibase.com/wp-content/uploads/2015/01/cadvisor_host_portal3.png)
+
+#### cAdvisor Mutli-Host Portal
+![cAdvisor Mutli-Host Portal](http://axibase.com/wp-content/uploads/2015/01/multi_host_portal3.png)
```

Comment:
Either these images need to be added to this repo, or you can drop them from this document and link to an external page (for more info...)